wiki/leave-insiders.md

```diff
@@ -0,0 +1,63 @@
+# Opting out of Windows Insider updates (Windows 11)
+
+If your machine is enrolled into the Windows Insider Program and you wish to stop receiving Insider updates, follow the steps below
+
+## Determine your current channel
+
+Open Settings, go to Windows Updates and Windows Insider Program
+
+![](img/leave-insiders/wu-page.png)
+
+There, look to the right side of "Choose your Insider settings" to determine your Insider channel, it can be Dev, Beta or Release Preview.
+
+![](img/leave-insiders/wu-insiders-channel.png)
+
+## For Beta/Release Preview
+
+If you found your channel to be Beta or Release Preview, opting out of Insider updates should be as easy as turning on a switch.
+
+Open the "Stop getting preview builds" section and turn on "Unenroll this device when the next version of Windows releases"
+
+![](img/leave-insiders/wu-insiders-stop.png)
+
+Restart your PC and you should be opted out.
+
+## For Dev
+
+If you have the Dev channel, you would want to pay attention to the build number, as it will affect whether you can leave Insiders without a clean install.
```

Comment:
```suggestion
If you are in Dev channel, you need to pay attention to the build number, as it will affect whether you can leave Insiders without a clean install.
```